zbus/src/message/mod.rs

```diff
@@ -18,9 +18,23 @@ use crate::OwnedFd;
 use crate::{
     utils::padding_for_8_bytes,
     zvariant::{DynamicType, EncodingContext, ObjectPath, Signature, Type},
-    EndianSig, Error, MessageBuilder, MessageField, MessageFieldCode, MessageFields, MessageHeader,
-    MessagePrimaryHeader, MessageType, QuickMessageFields, Result, MIN_MESSAGE_SIZE,
-    NATIVE_ENDIAN_SIG,
+    Error, Result,
+};
+
+mod builder;
+pub use builder::MessageBuilder;
+
+mod field;
+pub use field::{MessageField, MessageFieldCode};
+
+mod fields;
+pub use fields::MessageFields;
```

Comment:
* with the new path, I really don't think we should ask user to type `Message` in the names if it's already part of the module. It should be `message::{Message, Builder, Field, FieldCode..}`.
* we need deprecation for top-level for `foo::Bar` (as agreed, I thought?).

Reply:
yes, this is not the latest series...